src/puppetlabs/puppetdb/http/query.clj

```diff
@@ -72,6 +118,18 @@
     req
     (restrict-query ["=" ["node" "active"] true] req)))
 
+(defn restrict-query-to-active-nodes'
```

Comment:
Yeah, the problem with have a -old or something like that is we then have to rename everywhere that uses it to point to the new name of the old thing. Then visit it again when we can switch it from the new name (old code) to the old name (new code). ' or * is usually how I've seen closely related functions named, but am open to other ideas. I think change existing stuff to a -old name or something just makes work for ourselves though.